app/src/main/AndroidManifest.xml

```diff
@@ -51,7 +51,8 @@
       android:theme="@style/OppiaThemeWithoutActionBar" />
     <activity
       android:name=".app.ongoingtopiclist.OngoingTopicListActivity"
-      android:theme="@style/OppiaThemeWithoutActionBar" />
+      android:theme="@style/OppiaThemeWithoutActionBar"
+      android:label="@string/ongoing_topic_list_activity_title"/>
```

Comment:
It requires reformatting, press ctrl + alt  + L to reformat.

Reply:
https://github.com/oppia/oppia-android/commit/533e8529ec1a10b22bee2e28420840aa75947f65
Change as requested.